www/www.reactos.org/testman/detail.php

```diff
@@ -53,7 +53,7 @@
 		$pattern_core = "#^([a-z]*:?\()([a-zA-Z0-9\/_]+.[a-z]+):([0-9]+)(\))#m";
 		$pattern_test = "#^([a-zA-Z0-9_]+.[a-z]+):([0-9]+)(: )#m";
 
-		$replacement_core = '$1<a href="' . VIEWVC_TRUNK . '/reactos/$2?revision=' . $row["revision"] . '&amp;view=markup#l$3">$2:$3</a>$4';
+		$replacement_core = '$1<a href="' . VIEWVC_TRUNK . '/blob/' . $row["revision"] . '$2#L$3">$2:$3</a>$4';
```

Comment:
No `blob/`, only `/`?

Reply:
No, because one refers to a file on GitHub as `/blob/<revision>/filename`.

Reply:
I actually changed it, because i defined blob in `config.inc.php` and it was a mistake to appentd `blob` twice
```
define("VIEWVC", "https://github.com/reactos/reactos");
define("VIEWVC_TRUNK", VIEWVC . "/blob");
```